src/domain/header/MainNavbar.js

```diff
@@ -98,6 +101,26 @@ class MainNavbar extends React.Component {
               </NavItem>
             </LinkContainer>
           </Nav>
+          <Nav pullRight>
+            <TabbableNavDropdown
+              as="li"
+              id="user-nav-dropdown"
+              renderToggle={(props) => {
+                return (
+                  <a {...props} className="app-MainNavbar__selection" href="#" type="button">
+                    {t('Accessibility settings')}
+                    <span className="app-MainNavbar__selection-overview">
+                      {`${(getFromLocalStorage(ACCESSIBILITY_LOCAL_STORAGE_KEY) || []).length} ${t('Selections')}`}
```

Comment:
> getFromLocalStorage(ACCESSIBILITY_LOCAL_STORAGE_KEY) || []

Could abstract this to its own utility function? Same for

> storeInLocalStorage(ACCESSIBILITY_LOCAL_STORAGE_KEY, selectedViewpoints)

in another component.

Reply:
Done